src/%IPM/Main.cls

```diff
@@ -258,95 +322,189 @@ run C:\Temp\MyCommands.json, where contents of the file are as follows:
 <modifier name="force" aliases="f" description="If specified, the module will be uninstalled even if other modules depend on it." />
 <modifier name="recurse" aliases="r" description="Also recursively uninstall dependencies. By default, will not uninstall dependencies that are also required by other installed modules; the -force flag overrides this." />
 <modifier name="quiet" aliases="q" dataAlias="Verbose" dataValue="0" description="Produces minimal output from the command." />
 <modifier name="verbose" aliases="v" dataAlias="Verbose" dataValue="1" description="Produces verbose output from the command." />
 <modifier name="purge" dataAlias="Purge" dataValue="1" description="Purge data from tables during uninstall." />
-<parameter name="module" required="true" description="Name of module to uninstall" />
+
 </command>
 
-<command name="cos">
-<description>Executes ObjectScript</description>
-<example>cos set $Namespace = "MYAPP"</example>
-<parameter name="cos" required="true" trailing="true" description="ObjectScript command(s) to execute" />
+<command name="cos" aliases="isc-os">
+<description>
+	Executes the provided ObjectScript expression.
+</description>
+
+<!-- Examples -->
+<example description="Run the Objectscript expression to set the namespace to MYAPP">
+	cos set $Namespace = "MYAPP"
+</example>
+
+<!-- Parameters -->
+<parameter name="expression" required="true" trailing="true" description="ObjectScript expression(s) to execute" />
+
 </command>
 
 <command name="orphans">
-<description>Lists resources in the current namespace's default code database that are not part of any module.</description>
-<example>orphans -type CLS</example>
+<description>
+	Lists resources in the current namespace's default code database that are not part of any module.
+</description>
+
+<!-- Examples -->
+<example description="List resources that are classes that are not part of any module in the current namespace">
+	orphans -type CLS
+</example>
+
+<!-- Modifiers -->
 <modifier name="type" aliases="t" value="true" description="Type (e.g., extension) of resource to show; if unspecified, all types are included." />
+
 </command>
 
 <command name="list-installed" aliases="list">
-<description>Lists modules installed in the current namespace</description>
-<example description="Shows all installed modules in tree format.">list-installed -tree</example>
+<description>
+	Lists modules installed in the current namespace.
+</description>
+
+<!-- Examples -->
+<example description="Shows all installed modules in tree format.">
+	list-installed -tree
+</example>
+
+<!-- Modifiers -->
 <modifier name="tree" aliases="t" description="If specified, show dependency tree for installed modules." />
-<modifier name="description" aliases="d" dataAlias="Desc" dataValue="1" description="Additional information is displayed for each module." />
```

Comment:
Why deleted?

Reply:
Thank you Dmitry and good catch! I miss delete it in a merge, now it's added back now!